test/fixtures/config-autoload/ts/array/postcss.config.cts

```diff
@@ -0,0 +1,23 @@
+import type { Config } from 'postcss-load-config';
+
+module.exports = function (api: Config) {
+  return {
+    parser: 'sugarss',
+    syntax: 'sugarss',
+    // FIXME: To confirm. Apparently there's no longer an available option `mode`?
```

Comment:
Same as https://github.com/webpack-contrib/postcss-loader/pull/632/files#r1155280217

Reply:
Is it not avaliable only in types?

Reply:
> Is it not avaliable only in types?

I am not sure. According to the documentation: https://github.com/postcss/postcss-load-config#options
I don't see the `mode` option.

I tried searching in their [codebase for options](https://github.com/postcss/postcss-load-config/blob/3f9f537e4eb8e861dde9fd83f52e83886dfdf909/src/options.js), but I can't find a mention of `mode` either. :thinking:

Reply:
We apply `mode` to our logic so developer can use the `mode` option from configuration

Reply:
I want to clarify, this one comes from the [webpack configuration - mode](https://webpack.js.org/configuration/mode/)?

I guess, in this case, if I am not wrong, I could just merge the types from the PostCSS Config and the webpack config.

Reply:
https://github.com/webpack-contrib/postcss-loader/pull/632/files#r1155291339